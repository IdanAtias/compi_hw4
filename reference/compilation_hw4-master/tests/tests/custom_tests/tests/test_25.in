int x = 0;
matrix m(1,1) = x;
