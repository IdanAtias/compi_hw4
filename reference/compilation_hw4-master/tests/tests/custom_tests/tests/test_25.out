x=
0
MatrixExpected
